udata/core/user/api.py

```diff
@@ -175,6 +176,28 @@ def get(self):
         return notifications
 
 
+@me.route('/apikey', endpoint='my_apikey')
+class ApiKeyAPI(API):
+    @api.secure
+    @api.doc('generate_apikey')
+    @api.marshal_with(apikey_fields)
+    @api.response(201, 'API Key generated')
+    def post(self):
+        '''(Re)Generate my API Key'''
+        current_user.generate_api_key()
```

Comment:
How long does it take? Is that a potential source of DoS? Should we threshold that API?